docs/УправлениеИнтеграциями.md

```
@@ -0,0 +1,428 @@
+
+КонецПроцедуры
+```
+
+#### SFTP
+
+Для обмена с SFTP подразумевается, что файл есть на SFTP-сервере. Для избежания ошибок пути исходящего и входящего каталогов должны быть со слэшом "/" в начале и конце.
+
+```bsl
+Функция ПолучитьФайлОбменаСSFTPСервера(ИнтеграционныйПоток, КаталогПоиска, СтруктураИстории)
+
+	ПараметрыСообщения = пбп_ИнтеграцииFTPSFTP.ПараметрыСообщенияFTPSFTP();
```

Markdownlint

Column: 1
Hard tabs

(MD010, no-hard-tabs)